gpkit/small_scripts.py

```diff
@@ -64,16 +64,16 @@ def mag(c):
         return c
 
 
-def unitstr(units, into="%s", options="~"):
+def unitstr(units, into="%s", options="~", dimless='-'):
```

Comment:
Yes -- `results_table` sets `dimless` to `""` -- nothing is printed for unitless quantities.

`unitstr` is being a little sloppy and probably needs documentation -- it's potentially confusing because it returns `into % units if units else ""`. This means that when `dimless` is `"-"` and `into` is `"[%s]"`, `unitstr` currently returns `"[-]"`, whereas when `dimless` is `""` and `into` is `"[%s]"`, `unitstr` currently returns `""`.

Documentation, suggestions, or improvements welcome...